src/test/ui/destructuring-assignment/tuple_struct_destructure_fail.stderr

```diff
@@ -49,6 +58,15 @@ LL |     SingleVariant(S, T)
 ...
 LL |     Enum::SingleVariant(_) = Enum::SingleVariant(1, 2);
    |     ^^^^^^^^^^^^^^^^^^^^^^ expected 2 fields, found 1
+   |
+help: use `_` to explicitly ignore each field
+   |
+LL |     Enum::SingleVariant(_, _) = Enum::SingleVariant(1, 2);
+   |                          ^^^
+help: use `..` to ignore all unmentioned fields
+   |
+LL |     Enum::SingleVariant(_, ..) = Enum::SingleVariant(1, 2);
+   |                          ^^^^
```

Comment:
I think that we should check that the present fields are all already placeholders, and if so go ahead and suggest `Enum::SingleVariant(..)` instead here.

Reply:
Done.